examples/csharp/helloworld-from-cli/Greeter/Greeter.csproj

```diff
@@ -1,18 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AssemblyTitle>Greeter</AssemblyTitle>
-    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
-    <DebugType>portable</DebugType>
-    <AssemblyName>Greeter</AssemblyName>
-    <PackageId>Greeter</PackageId>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Protobuf" Version="3.5.0" />
-    <PackageReference Include="Google.Protobuf.Tools" Version="3.5.0" />
-    <PackageReference Include="Grpc" Version="1.8.0" />
-    <PackageReference Include="Grpc.Tools" Version="1.8.0" />
+    <PackageReference Include="Grpc" Version="1.10.0" />
```

Comment:
We can't make the example depend on a non-existent version of Grpc (even though the grpc.io page leads our examples in the release branch, I assume many users are looking for examples in the master branch.

Here's what I'd suggest:
1. Let's create a separate PR for changes to the examples (assuming that the "new" Grpc.Tools package already exists).
2. Let's remove all the changes to example from this PR and focus on just the changes to Grpc.Tools

The examples and Grpc.Tools are in a way independent - once can easily generate the Grpc.Tools nugets from one PR and then test it on modified examples in the other PR.
The example PR should be only merged once the first modified version of Grpc.Tools is actually released.

Reply:
Excellent point, I did not think about it! The version 1.10 is released a source actually before the package is published. So the examples PR must be merged after the package is out.